ui/src/pages/Login.tsx

```diff
@@ -1,14 +1,76 @@
 import * as React from "react";
 import { Redirect } from "react-router-dom";
-
+import { makeStyles } from "@material-ui/core/styles";
+import Card from "@material-ui/core/Card";
+import CardActions from "@material-ui/core/CardActions";
+import CardContent from "@material-ui/core/CardContent";
+import Button from "@material-ui/core/Button";
+import Typography from "@material-ui/core/Typography";
+import styled, { css } from "styled-components";
 import { useAuth } from "../hooks/auth";
 
-export const Login: React.FC = () => {
+const CardDiv = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  width: 100vw;
+  position: absolute;
+`;
+
+const StyledCard = styled(Card)`
+  max-height: 35vh;
+  max-width: 25vw;
+`;
+
+const TitleText = styled.p`
+  font-size: 20px;
```

Comment:
can you move the TitleText and Message Text to the typography file? And then use fontSize.whicheverText for consistency.

Reply:
Wait nvm I realized this is all lofi so the font sizes are still in the air - would be good though to still keep the message and title texts in the typography file like the other components :)